holmes.cabal

```diff
@@ -87,13 +87,14 @@ test-suite test
   main-is:  Main.hs
 
   build-depends: base
-               , containers >= 0.6 && < 0.7
-               , hashable >= 1.3 && < 1.4
-               , hedgehog >= 1.0 && < 1.1
+               , containers >= 0.6 && < 0.8
+               , hashable >= 1.3 && < 1.5
+               , hedgehog >= 1.0 && < 1.5
+               , hspec >= 2.7 && < 2.12
                , holmes
-               , primitive >= 0.7 && < 0.8
-               , transformers >= 0.5 && < 0.6
-               , tasty >= 1.2 && < 1.5
+               , primitive >= 0.7 && < 0.10
+               , transformers >= 0.5 && < 0.6.2
+               , tasty >= 1.2 && < 1.6
                , tasty-discover
```

Comment:
Potentially unnecessary if the test suite code isn't directly calling into this code; the `build-tool-depends` is already here and might be more appropriate.